pep-0683.rst

```diff
@@ -245,9 +272,107 @@ would be noticed.
 Again, the only realistic way that the magic refcount would be reached
 (and then reversed) is if it were done deliberately.  (Of course, the
 same thing could be done efficiently using ``Py_SET_REFCNT()`` though
-that would be even less of an accident.) At that point we don't
+that would be even less of an accident.)  At that point we don't
 consider it a concern of this proposal.
 
+On 32-bit builds it isn't so obvious.  The magic refcount would be 2^28.
+Using the same specs as above, it would take roughly 1 second to
+accidentally immortalize an object.  Under reasonable conditions, it
+is still highly unlikely that an object be accidentally immortalized.
+It would have to meet these criteria:
+
+* targeting a non-immortal object (so not one of the high-use builtins)
+* the extension increfs without a corresponding decref
+  (e.g. returns from a function or method)
+* no other code decrefs the object in the meantime
+
+Under those conditions it would reach accidental immortality (on 32-bit)
+in, at most, a year if it averaged at least one of those increfs every
+158 seconds on that hypothetical workstation.  Of course, then it would
```

Comment:
Is there explanation for the precise value of 158 seconds?

Reply:
It's arithmetic using the specified details so I left it out.

Reply:
The fixed value of one year? I suppose I don't understand the example, as earlier the PEP talks to the fastest possible cycles (one reference change per CPU cycle) -- you then list three conditions from lines 284 to 287 inclusive to qualify the hypotethetical scenario, but I don't follow the jump to one year and corresponding 158 seconds.

Reply:
Hmm, I'm not sure I gave the right number (close enough).  However, the example isn't helpful enough to make it worth including an explanation.  So I'll remove it.